airflow/cli/commands/dag_command.py

```diff
@@ -503,6 +503,5 @@ def dag_reserialize(args, session: Session = NEW_SESSION):
     session.query(SerializedDagModel).delete(synchronize_session=False)
 
     if not args.clear_only:
-        dagbag = DagBag()
-        dagbag.collect_dags(only_if_updated=False, safe_mode=False)
```

Comment:
Does this line need to be kept?

Reply:
No, and I described in the PR description why it is not needed.

Reply:
But the arguments here is different from the call in `__init__`.

Reply:
Since `DagBag().file_last_changed` will be empty, I _think_ it won't make a difference if `only_if_updated` is set to `True` or `False`

Reply:
only_if_update = False means the dags will be read again if the file has changed.  In our case, we don't need to load the same files twice, so this parameter doesn't apply here.